common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -148,13 +148,16 @@ def setUp(self):
         super(DiscussionTabSingleThreadTest, self).setUp()
         self.discussion_id = "test_discussion_{}".format(uuid4().hex)
 
-        # Create a course to register for
-        CourseFixture(**self.course_info).install()
+        course = CourseFixture(**self.course_info)
```

Comment:
Nit: I would be tempted to create a base test class and put this common setUp code in its setUp method.